server/tests/events.test.js

```diff
@@ -0,0 +1,162 @@
+        .send({
+            eventName: 'how dinosaurs helped establish modern architecture'
+        })
+    expect(res.status).toEqual(403)
+})
+
+describe('alumni can successfully create new events', () => {
+
+    it("all parameters are successfully saved on the event", async () => {
+        var alumnus = await alumniSchema.findOne()
+        var user = await userSchema.findById(alumnus.user)
+        var date = new Date()
+        var eventRequestObj = {
+            name: 'staging the mars landing',
+            years: [ 2012, 2013, 2014 ],
+            link: 'linkthelink.com',
+            school: alumnus.school,
+            description: "they staged the moon. we do mars. looking for innovators, artists, people who want a hand in making history!",
+            date
+        }
+        var res = await request(app).post(`/events/create/${alumnus.user}`).send(eventRequestObj)
+        var resJson = JSON.parse(res.text)
+        var createdEvent = await eventSchema.findById(resJson.event._id)
+        expect(createdEvent.creator).toEqual(alumnus.user)
+        expect(createdEvent.name).toEqual(eventRequestObj.name)
+        expect(createdEvent.date).toEqual(eventRequestObj.date)
+        expect(createdEvent.description).toEqual(eventRequestObj.description)
+        expect(createdEvent.link).toEqual(eventRequestObj.link)
+        expect(createdEvent.school).toEqual(eventRequestObj.school)
+        expect(JSON.stringify(createdEvent.years)).toEqual(JSON.stringify(eventRequestObj.years))
+        expect(res.status).toEqual(200)
+    })
+
+    it('if no school provided use alumni school as default', async () => {
+        var alumnus = await alumniSchema.findOne()
+        var user = await userSchema.findById(alumnus.user)
+        var date = new Date()
+        var eventRequestObj = {
+            name: 'half-sour enthusiast networking network',
+            years: [ 2012, 2013, 2014 ],
+            link: 'linkthesink.com',
+            description: 'hoping to create a space for fellow half-sour pickle enthusiasts. full-sour speculators welcome.',
+            date
+        }
+        var res = await request(app).post(`/events/create/${alumnus.user}`).send(eventRequestObj)
+        expect(res.status).toEqual(200)
+        var resJson = JSON.parse(res.text)
+        var createdEvent = await eventSchema.findById(resJson.event._id)
+        expect(createdEvent.school).toEqual(alumnus.school)
+    })
+})
+
+describe('alumni can only create events within their access context', () => {
+
+    beforeAll(async () => {
+        // create alumnus
+        await request(app).post('/util/addAlumni/').send({
+            email: "schoolalum@ofi.com",
+            name: "school alum",
+            accessContexts: [ "INTRASCHOOL" ],
+            gradYear: Math.random(),
+            country: "Germany",
+            city: "Osgiliath"
+        })
+        var alumnus = await alumniSchema.findOne({ name: "school alum", city: "Osgiliath" })
```

Comment:
I like what you did here, wondering if maybe this could be available as part of the initial seed data I'm imagining a specific alum/school that we can access with certain details would probably be helpful for a lot of different tests